src/labs/requests/NewLabRequest.tsx

```diff
@@ -24,10 +24,12 @@ const NewLabRequest = () => {
   const [newLabRequest, setNewLabRequest] = useState({
     patient: '',
     type: '',
-    notes: '',
+    // notes: [''],
```

Comment:
Just remove this line

Reply:
My bad.

Reply:
I did it as requested but looks like the deployment failed and I don't really know if it was my fault.